CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ x.y.z Release notes (yyyy-MM-dd)
 * None.
 
 ### Fixed
+* Fix a runtime crash which happens in some Xcode version (Xcode < 12, reported in Xcode 12.5), where SwiftUI is not weak linked by default. This fix only works for Cocoapods projects.
```

Comment:
Are they ways to get the crash through methods other than Cocoapods?

Reply:
yep, there is a reported case of someone getting the same error using SPM, in that case the solution is to add "SwiftUI as -weak_framework" to "Other linker flags" OTHER_LDFLAGS to the project importing the module

Reply:
I supposed this is an Xcode issue (that happens sometimes), and not related to which dependency manager the user is using